server/app/app.go

```diff
@@ -287,7 +281,22 @@ func registerSPAHandler(router *mux.Router, cfg httpServerConfig, analyticsEnabl
 				Env,
 			),
 		)
+}
 
+func registerOtlpServer(app *App, testDB model.Repository) {
+	exporter := otlp.NewExporter(testDB)
+	grpcOtlpServer := otlp.NewGrpcServer(":4318", exporter)
+	httpOtlpServer := otlp.NewHttpServer(":4317", exporter)
```

Comment:
It's the other way around: 
```suggestion
	grpcOtlpServer := otlp.NewGrpcServer(":4317", exporter)
	httpOtlpServer := otlp.NewHttpServer(":4318", exporter)
```